packages/theme-data/src/colorSchemes/darkBlue/components/slider.js

```diff
@@ -0,0 +1,25 @@
+export default {
+  // Focused
+  "slider.focused.halo.color": {
+    value: { ref: "basics.colors.autodeskBlue400" }
```

Comment:
This can reference the themes `colorScheme` instead like above as this maps to `colorScheme.accentColor`.